loculus_values/environment_specific_values/production.yaml

```diff
@@ -88,20 +88,20 @@ resources:
       memory: "10Gi"
   keycloak:
     requests:
-      memory: "500Mi"
+      memory: "3Gi"
       cpu: "1000m"
     limits:
       memory: "3Gi"
   silo:
     requests:
-      memory: "100Mi"
-      cpu: "1000m"
+      memory: "500Mi"
+      cpu: "500m"
```

Comment:
Idk, I assume that these don't require much CPU except when they do when they're being heavily used (e.g. once someone starts using them as an API). To me giving 1cpu each in prod doesn't seem excessive.

Reply:
We'd be having 8 CPUs idle just for LAPIS/SILO, just wondering whether you're aware of that. If you are fine with me.

Reply:
> We'd be having 8 CPUs idle just for LAPIS/SILO, just wondering whether you're aware of that.

To be pedantic, that's not true in my understanding. We would be saying that production LAPIS/SILO would be guaranteed 8 CPUs if it needed them. When this CPU isn't being used by LAPIS/SILO it can be used by other stuff, e.g. preview instances (i.e. the difference between the preview instance's `requests` and their `limit`).

Reply:
Ah yeah you're right, sorry. However it does mean we do require a minimum number of CPUs - but that's fine.